log-patch.diff

```diff
@@ -0,0 +1,129 @@
+diff --git a/config.h b/config.h
```

Comment:
What's up with this patch file? Git already keeps track of changes so what's the point in having it here?